src/azure-cli/azure/cli/command_modules/rdbms/tests/latest/test_rdbms_commands.py

```diff
@@ -101,29 +101,53 @@ def _test_server_mgmt(self, database_engine, resource_group_1, resource_group_2)
         skuname = 'GP_{}_{}'.format(family, old_cu)
         newskuname = 'GP_{}_{}'.format(family, new_cu)
         loc = 'eastus'
+        minimal_tls_version = 'TLS1_1'
+        public_network_access = 'Disabled'
 
         geoGeoRedundantBackup = 'Disabled'
         geoBackupRetention = 20
         geoloc = 'eastus'
 
-        # test create server
-        self.cmd('{} server create -g {} --name {} -l {} '
-                 '--admin-user {} --admin-password {} '
-                 '--sku-name {} --tags key=1 --geo-redundant-backup {} '
-                 '--backup-retention {}'
-                 .format(database_engine, resource_group_1, servers[0], loc,
-                         admin_login, admin_passwords[0], skuname,
-                         geoRedundantBackup, backupRetention),
-                 checks=[
-                     JMESPathCheck('name', servers[0]),
-                     JMESPathCheck('resourceGroup', resource_group_1),
-                     JMESPathCheck('administratorLogin', admin_login),
-                     JMESPathCheck('sslEnforcement', 'Enabled'),
-                     JMESPathCheck('tags.key', '1'),
-                     JMESPathCheck('sku.capacity', old_cu),
-                     JMESPathCheck('sku.tier', edition),
-                     JMESPathCheck('storageProfile.backupRetentionDays', backupRetention),
-                     JMESPathCheck('storageProfile.geoRedundantBackup', geoRedundantBackup)])
+        if database_engine == 'mariadb':
+            # test create server
+            self.cmd('{} server create -g {} --name {} -l {} '
+                        '--admin-user {} --admin-password {} '
+                        '--sku-name {} --tags key=1 --geo-redundant-backup {} '
+                        '--backup-retention {}'
+                        .format(database_engine, resource_group_1, servers[0], loc,
+                        admin_login, admin_passwords[0], skuname,
+                        geoRedundantBackup, backupRetention),
+                        checks=[
+                            JMESPathCheck('name', servers[0]),
+                            JMESPathCheck('resourceGroup', resource_group_1),
+                            JMESPathCheck('administratorLogin', admin_login),
+                            JMESPathCheck('sslEnforcement', 'Enabled'),
+                            JMESPathCheck('tags.key', '1'),
+                            JMESPathCheck('sku.capacity', old_cu),
+                            JMESPathCheck('sku.tier', edition),
+                            JMESPathCheck('storageProfile.backupRetentionDays', backupRetention),
+                            JMESPathCheck('storageProfile.geoRedundantBackup', geoRedundantBackup)])
```

Comment:
Please add test for default value check for tsl_version and public_network_access here.

Reply:
Done